docs/LANGUAGE_TRANSLATION.md

```diff
@@ -2,19 +2,25 @@
 
 > **Note:** development on this project is moving quickly, so parts of this document may be out of date. Feel free to ask questions in the `#tc-aid-translations` channel in Slack.
 
-1. [Overview](#Overview)
-   1. [The translation spreadsheet](#the-translation-spreadsheet) (**DEPRECATED**)
-   2. [Translation files](#translation-files)
-   3. [Special links](#special-links)
-2. [Maintaining translations](#maintaining-translations)
-   1. [Translating existing terms](#translating-existing-terms)
-   2. [Creating a new term](#creating-a-new-term)
-   3. [Adding a new language](#adding-a-new-language)
-3. [Technical details](#technical-details)
-   1. [Adding a term in markup (`data-translation-id`)](#adding-a-term-in-markup-data-translation-id)
-   2. [Getting a translated term in js code (`get`)](#getting-a-translated-term-in-js-code-get)
-   3. [Setting the current language (`language`)](#setting-the-current-language-language)
-   4. [Running translation substitution (`translate`)](#running-translation-substitution-translate)
+- [Language Translation](#language-translation)
+  - [Overview](#overview)
+    - [The translation spreadsheet](#the-translation-spreadsheet)
+    - [Translation files](#translation-files)
+    - [Special links](#special-links)
+  - [Maintaining Translations](#maintaining-translations)
+    - [Translating existing terms](#translating-existing-terms)
+    - [Creating a new term](#creating-a-new-term)
+    - [Adding a new language](#adding-a-new-language)
+      - [Create a new translation file](#create-a-new-translation-file)
+      - [Adding a new flag image](#adding-a-new-flag-image)
+      - [Moment.js](#momentjs)
+      - [Enabling the language](#enabling-the-language)
+      - [Add language checklist](#add-language-checklist)
+  - [Technical Details](#technical-details)
+    - [Adding a term in markup (`data-translation-id`)](#adding-a-term-in-markup-data-translation-id)
+    - [Getting a translated term in js code (`get`)](#getting-a-translated-term-in-js-code-get)
+    - [Setting the current language (`language`)](#setting-the-current-language-language)
+    - [Running translation substitution (`translate`)](#running-translation-substitution-translate)
```

Comment:
Hmmm, seems this update happened automatically?

Reply:
@mc-funk I think it's being an actual table of contents... in my IDE it says `Table of Contents (up to date)` like this, and on master it says `Table of Contents (up to date)` - I think I have a formatter in my IDE that probably made it up to date so should be okay!